### ISSUE TITLE
Brainstorm for New Survey: Contribution Experience 

### ISSUE THREAD

**Issue by AnaMMedina21** [open-telemetry/sig-end-user#42]
Thinking we should find out what folk's current experience is with contributing, this will let us know how we can make it better and help us recruit new members

Will fill out more details later

**Comment by avillela** [comment]
One area where this applies and is probably the lowest barrier to entry is docs, so I would love thoughts from @open-telemetry/docs-approvers on this!

We might want to tap some SIG maintainers here too. Are there any SIGs that we know of that need some extra contribution love? This may be a question for @open-telemetry/governance-committee?

**Comment by svrnm** [comment]
Please take a look at https://github.com/open-telemetry/community/pull/2162 as well, since I suggested creating a SIG for contributor experience, so this could be a coordinated effort eventually. This should not stop you from creating such a survey and I agree (very selfishly) that docs is a good starting point

**Comment by jpkrohling** [comment]
I agree -- the survey is a good idea to have, and we could even consider having it _regularly_. We have a list of contributors to the project via devstats, so we could target people from that list.

**Comment by avillela** [comment]
I like the idea of doing a survey regularly. Can you point me to how we can find the contributors via devstats?

**Comment by jpkrohling** [comment]
This could be a good start: 

https://opentelemetry.devstats.cncf.io/d/52/new-contributors-table?orgId=1

**Comment by avillela** [comment]
This is great! Thanks for sharing!!

**Comment by avillela** [comment]
Related PR: https://github.com/open-telemetry/community/pull/2162

**Comment by danielgblanco** [comment]
A combined effort on this would be amazing. I think it's possible for organization admins to see emails of all members as well (thinking about automating this), although that will be a subset of contributors.

Would the intention be to have an "onboarding survey" to see how new contributors found their experience?

**Comment by avillela** [comment]
> Would the intention be to have an "onboarding survey" to see how new contributors found their experience?

Yeah, exactly - onboarding specifically around OTel contributions.

**Comment by AnaMMedina21** [comment]
Hey @open-telemetry/sig-contributor-experience-maintainers, the End-User SIG had brainstormed coming up with a survey to understand the contributor experience and ways to improve it. With the creation of the new SIG we feel like this fits your wheelhouse more and wanted to put it on your radar, we are happy to help in any way

**Comment by svrnm** [comment]
should this issue be moved then? or should we have 2 of them? End User SIG has so much more experience in running those surveys and accomplishing reach, so I would prefer to collaborate on that still

**Comment by svrnm** [comment]
A thought I wanted to have written down: it would be interesting to see if we can split this survey into two: a contributor experience survey and a non-contributor experience survey where the second is aimed towards people that wanted to contribute but failed to do so or stopped contributing. This would be helpful to identify some issues on how people get into our community

**Comment by danielgblanco** [comment]
Regarding two surveys... I think we can get into survey fatigue (although I guess we wouldn't be targeting the same groups). I think it would be possible to create a survey with an initial condition (have you contributed before) and go to a different section as explained in https://www.howtogeek.com/437542/how-to-create-a-google-form-with-branching-logic/.

That way only one survey may come out, with a message like "have you contributed to OTel, or have thought about it didn't get to it". This may be easier to handle and aggregate responses (some of them may be similar questions for both groups)

**Comment by mx-psi** [comment]
I gave this a try here: https://docs.google.com/document/d/1YlwHJlJjuaR4i8VqWf1eIOpT158TOHdLii8iC0ndJXc/edit?tab=t.0#heading=h.wkyobsrv4abk My focus has been to also ask questions related to existing community contributors and not only newbies.

Note that we have a duplicate of this issue here: open-telemetry/sig-contributor-experience/issues/27. I like this one better, should we close the other one and transfer this one to the sig-contributor-experience repo?

**Comment by AnaMMedina21** [comment]
@mx-psi Survey is looking good! Do yall need our help or would yall like to run it yourselves? We are super happy to help

**Comment by mx-psi** [comment]
I have turned the Google Doc into a Google Form and I am looking for volunteers to do a dry-run and give feedback about it. If you are interested please reach out to me in the CNCF Slack.

If all goes well, I would like to release this on the week of January 6th and use the help from the End-User SIG to share this :)

**Comment by svrnm** [comment]
@mx-psi happy to take a look

**Comment by mx-psi** [comment]
@svrnm Great! Sent you a DM on the CNCF Slack

**Comment by mx-psi** [comment]
Survey is out! https://opentelemetry.io/blog/2025/contribex-survey/

**Comment by danielgblanco** [comment]
Let's keep this issue open. We still have to create an issue template with a survey checklist, but we've got this from another survey

https://github.com/open-telemetry/sig-end-user/issues/69#issue-2573741202

I'll be good to keep this open until those steps are finished for this. @AnaMMedina21 I will modify the description above to match that.

**Comment by mx-psi** [comment]
> Do we need to promote this on socials apart from sharing the blogpost, which I imagine will happen regardless?

Let's discuss that on the thread here: https://cloud-native.slack.com/archives/C01RT3MSWGZ/p1736418871791729. I think it makes sense to have a weekly message, or whatever strategy you have pursued in the past for surveys.

> Also, do we need a banner on the website?

There is currently a banner for it on [the front page](https://opentelemetry.io/). We have too many banners, so @svrnm was going to talk about this with the Comms SIG.

**Comment by reese-lee** [comment]
Update: They are in the process of writing up a blog summary based on their results. Will follow up in another week to check the progress. ([Slack thread](https://cloud-native.slack.com/archives/C06TMJ2R0SK/p1741625585436349) for reference.)

**Comment by danielgblanco** [comment]
@mx-psi did ContribEx SIG get to start on that? I assume it's been hectic as usual after KubeCon.

**Comment by mx-psi** [comment]
I am brainstorming what questions to try to explore in the dataset with @tiffany76. If anybody else wants to contribute I would be happy to work with them, I am going to be a bit slow in making progress on this one.

**Comment by AndrejKiri** [comment]
@mx-psi I will be happy to help you and @tiffany76. Will DM you folks in Slack. 💪

**Comment by AndrejKiri** [comment]
Last week a [blog post](https://opentelemetry.io/blog/2025/contribex-survey-results/) has been published.

**Comment by AndrejKiri** [comment]
I added a link to the blog post to the survey README file: https://github.com/open-telemetry/sig-end-user/pull/138/

Closing this issue because the work has been finished and there is nothing further planned. Thanks all! 🙌

FYI @mx-psi @tiffany76
